peachjam/templates/account/password_change.html

```diff
@@ -0,0 +1,64 @@
+{% extends "user_account/layout.html" %}
+{% load i18n %}
+{% block title %}
+  {% trans "Change Password" %}
+{% endblock %}
+{% block account-tabs %}
+  {% include 'user_account/_tabs.html' with active='password' %}
+{% endblock %}
+{% block account-content %}
+  <form method="post" action="{% url 'account_change_password' %}">
```

Comment:
This form and the user name form can be a little smaller, Perhaps half the page instead of the full page.